src/client/apps/edit/components/header/yoast/index.tsx

```diff
@@ -0,0 +1,249 @@
+import { Box, color, Flex, space } from "@artsy/palette"
+import { avantgarde } from "@artsy/reaction/dist/Assets/Fonts"
```

Comment:
we should move away from using `avantgarde` and use `import { sans } from "@artsy/palette"` instead as it's the same font